scripts/versionAndPush.sh

```diff
@@ -2,6 +2,13 @@
 # exit upon error (DON'T -x !!)
 set -e
 
+if ! [[ $TRAVIS_COMMIT_MESSAGE =~ "[release]" ]]; then
```

Comment:
```suggestion
if ! [[ $TRAVIS_COMMIT_MESSAGE" =~ "chore: release" ]]; then
```
maybe? then it'll be compatible with angular rules

Reply:
maybe some regex `chore: .*[release].*` (not sure about syntax)

Reply:
The CI/CD version bump commit is called `chore(release): publish new package versions [skip ci]`.
We have to make a commit before this commit that triggers it. Preferably they look a bit different, but it should maybe be clear the first triggered the last. I agree we can require a certain prefix

Reply:
However, sometimes you may want to use a squash commit and also trigger a build, like:
```
feat(assets): Add endpoint [release]
```

Reply:
agree